configuration/external_parameters.rst

```diff
@@ -80,24 +40,86 @@ You can now reference these parameters wherever you need them.
 
         <doctrine:config>
             <doctrine:dbal
-                driver="pdo_mysql"
-                dbname="symfony_project"
-                user="%database.user%"
-                password="%database.password%"
+                host="%env(DATABASE_HOST)%"
             />
         </doctrine:config>
 
     .. code-block:: php
 
         $container->loadFromExtension('doctrine', array(
             'dbal' => array(
-                'driver'   => 'pdo_mysql',
-                'dbname'   => 'symfony_project',
-                'user'     => '%database.user%',
-                'password' => '%database.password%',
+                'host' => '%env(DATABASE_HOST)%',
             )
         ));
 
+You can also give the ``env(...)`` parameters a default value: the default value
+will be used whenever the corresponding environment variable is *not* found:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        #app/config/parameters.yml
+
+        parameters:
+            env(DATABASE_HOST): localhost
+
+    .. code-block:: xml
+
+        <?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
We usually simply replace the file extension. So here we would have `app/config/parameters.yml`, `app/config/parameters.xml`, and `app/config/parameters.php`. By the way, can you also add similar comments in the example above to, but using `config.yml`, `config.xml`, and `config.php` there?